requirements.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - jinja2>=2.10
   - jupyter_client>=6.1
   - jupyter_core>=4.6.0
-  - notebook>=6.1.0
+  - notebook>=6.1.0,<6.1.6
```

Comment:
I've created https://github.com/jupyter/enterprise_gateway/issues/922 so we can remember to remove (or bump) this pin once resolved.

Reply:
Thank you